ebos/eclgenericproblem_impl.hh

```diff
@@ -210,7 +212,8 @@ readRockParameters_(const std::vector<Scalar>& cellCenterDepths,
             if (!rockTableIdx_.empty()) {
                 tableIdx = rockTableIdx_[elemIdx];
             }
-            overburdenPressure_[elemIdx] = overburdenTables[tableIdx].eval(cellCenterDepths[elemIdx], /*extrapolation=*/true);
+            overburdenPressure_[elemIdx] =
+                overburdenTables[tableIdx].eval(cellCenterDepths[this->getLookUpData(elemIdx)], /*extrapolation=*/true);
```

Comment:
I think this should not be changed as we probably want the cell center of the cell on the leaf grid.

Reply:
If I understand correctly, `overburdenPressure_` has size `numElem` which is total amount of cells in the leaf grid view. To define the value of `overburdenPressure_` in certain leaf grid view cell with index `elemIdx`,  you look at` overburdenTables[/*tableIdx*/0].eval(cellCenterDepths[/*certain cell index*/], /*extrapolation*/true)`. Now, if these tables store information of the unrefined grid, then shouldn't /*certain cell index*/ be the index of the origin cell of the leaf-grid-view cell? Am I wrong? I do not know (yet) what eval() does though. 

To phrase it differently, if `overburdenTables[0] `stores information of the unrefined grid, does `eval(cellCenterDepths(**leaf-grid-view-cell-index**), true)` give certain level-0-cell-index to access data in the tables?

Reply:
Thanks!